couch/tests/test_couchv2.py

```diff
@@ -192,11 +194,15 @@ def test_check_without_names(aggregator, gauges):
         for gauge in gauges["cluster_gauges"]:
             aggregator.assert_metric(gauge, tags=expected_tags)
 
-        for gauge in gauges["erlang_gauges"]:
-            aggregator.assert_metric(gauge)
+    for gauge in gauges["erlang_gauges"]:
+        aggregator.assert_metric(gauge)
 
-        for gauge in gauges["replication_tasks_gauges"]:
-            aggregator.assert_metric(gauge)
+    for gauge in gauges["replication_tasks_gauges"]:
+        aggregator.assert_metric(gauge)
```

Comment:
It does not make sense for these to be inside the loop since they're not using the loop variable to assert tags